app/helpers/measures_helper.rb

```diff
@@ -61,4 +61,48 @@ def control_line_wrapping_in_duty_expression(expression)
       safe_join components.flatten.compact, ' '
     end
   end
+
+  def format_measure_condition_requirement(condition)
+    case condition.measure_condition_class
```

Comment:
We're asking a lot of the condition in this method. Should we move some of the logic around presentation to the model?

Reply:
You could also ask the measure condition class whether it's a threshold

```suggestion
    if condition.measure_condition_class.threshold?
```